clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesJob.groovy

```diff
@@ -53,6 +54,8 @@ class KubernetesJob implements Job, Serializable {
   @Override
   JobState getJobState() {
     job.status.succeeded == job.spec.completions ? JobState.Succeeded :
-      job.status.active > 0 ? JobState.Running : JobState.Unknown  // TODO(lwander) figure out how to extract failure state
+      job.status.active > 0 ? JobState.Running :
```

Comment:
Some text here to explain the possible paths through this big conditional might be helpful.